### PR TITLE
NLP examples notebook

### PR DESCRIPTION
**Pull request by piotrm0**
NLP input attribution example notebook based on a model from hugging face. Utilities to make that possible/convenient to implement and to visualize.

Also included a few things to handle backend detection and pytorch device specification issues.

### CODE REVIEW

File: trulens/visualizations.py [hunk 1]
```diff
@@ -999,3 +1012,301 @@ def __call__(
             attrs_input, x, output_file, blur, threshold, masked_opacity,
             combine_channels
         )
+
+
+class Output(ABC):
```
Comment (rshih32) [inline trulens/visualizations.py:1017]: nit:TextOutput
This looks like it deals specifically with text

File: trulens/visualizations.py [hunk 2]
```diff
@@ -999,3 +1012,301 @@ def __call__(
...
+
+    @abstractmethod
+    def render(self, s: str) -> str:
+        ...
+
+
+# TODO(piotrm): implement a latex output format
+
+
+class Text(Output):
```
Comment (rshih32) [inline trulens/visualizations.py:1052]: nit: if we are saying Output is TextOutput, maybe we call this PlainText

**Review by rshih32 — APPROVED** [review]

File: trulens/visualizations.py (hunk 1 above)
Comment (piotrm0) [reply trulens/visualizations.py:1017]: Don't mean it to handle text only but that is what is available presently.

File: trulens/visualizations.py (hunk 2 above)
Comment (piotrm0) [reply trulens/visualizations.py]: Renamed to PlainText .

**Pull request merged**